src/api/services/participantsService.ts

```diff
@@ -88,30 +71,30 @@ export const getAttachedSiteIDs = async (): Promise<SiteIdType[]> => {
 
 export const addSharingParticipants = async (
   participantSiteId: number,
-  siteIds: number[]
-): Promise<Participant[]> => {
+  siteIds: number[],
+  types: string[]
```

Comment:
nit: should we use clientType[]?

Reply:
To use ClientType in the service, I believe I would need to re-define the client types in the router as an enum like this:
```
  const ClientType = z.enum(['DSP', 'ADVERTISER', 'DATA_PROVIDER', 'PUBLISHER']);

  const sharingRelationParser = z.object({
    newParticipantSites: z.array(z.number()),
    newTypes: z.array(ClientType),
  });
```

Does that sound right to you? Or is there a better way?

Source: https://zod.dev/

Reply:
I think this looks better than string[]

Reply:
cool will do. There is also a bunch of uses of `string[]` in the components which I will update to use `ClientType[]`

Reply:
done!